tests/test_smoke.py

```
@@ -0,0 +1,142 @@
+        },
+        width=100,
+        height=100,
+    )
+
+    agent = AgentS3(
+        worker_engine_params={"engine_type": "mock", "model": "gpt-4o"},
+        grounding_agent=grounding,
+        platform="linux",
+    )
+
+    info, actions = agent.predict(
```

Ruff (0.14.10)

Unpacked variable `info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)